tagger/firmware/hls4ml_profile.py

```diff
@@ -0,0 +1,138 @@
+from argparse import ArgumentParser
+import os, shutil, json
+
+#Import from other modules
+from tagger.data.tools import make_data, load_data, to_ML
+from tagger.firmware.hls4ml_convert import convert
+import tagger.train.models
+from tagger.plot.makeEmulationPlot import plot_2d
+
+#Third parties
+import numpy as np
+import tensorflow as tf
+import tensorflow_model_optimization as tfmot
+import hls4ml
+from qkeras.utils import load_qmodel
+import mplhep as hep
+from sklearn.metrics import roc_curve, auc,precision_recall_curve
+import matplotlib.pyplot as plt
+
+
+
+# Setup plotting to CMS style
+hep.cms.label()
+hep.cms.text("Simulation")
+plt.style.use(hep.style.CMS)
+
+SMALL_SIZE = 20
+MEDIUM_SIZE = 25
+BIGGER_SIZE = 35
+
+LEGEND_WIDTH = 20
+LINEWIDTH = 3
+MARKERSIZE = 20
+
+colormap = "jet"
+
+plt.rc('font', size=SMALL_SIZE)          # controls default text sizes
+plt.rc('axes', titlesize=BIGGER_SIZE)    # fontsize of the axes title
+plt.rc('axes', labelsize=BIGGER_SIZE)    # fontsize of the x and y labels
+plt.rc('axes', linewidth=LINEWIDTH+2)              # thickness of axes
+plt.rc('xtick', labelsize=MEDIUM_SIZE)    # fontsize of the tick labels
+plt.rc('ytick', labelsize=MEDIUM_SIZE)    # fontsize of the tick labels
+plt.rc('legend', fontsize=MEDIUM_SIZE-2)            # legend fontsize
+plt.rc('figure', titlesize=BIGGER_SIZE)  # fontsize of the figure title
+
+
+import matplotlib
+
+matplotlib.rcParams['xtick.major.size'] = 20
+matplotlib.rcParams['xtick.major.width'] = 5
+matplotlib.rcParams['xtick.minor.size'] = 10
+matplotlib.rcParams['xtick.minor.width'] = 4
+
+matplotlib.rcParams['ytick.major.size'] = 20
+matplotlib.rcParams['ytick.major.width'] = 5
+matplotlib.rcParams['ytick.minor.size'] = 10
+matplotlib.rcParams['ytick.minor.width'] = 4
```

Comment:
We should refactor this in the future, but probably not needed for the current PR